contracts/components/staking/FortaStaking.sol

```diff
@@ -507,11 +551,12 @@ contract FortaStaking is BaseComponentUpgradeable, ERC1155SupplyUpgradeable, Sub
      * @notice Release reward owed by given `account` for its current or past share for a given `subject`.
      * @dev If staking from a contract, said contract may optionally implement ERC165 for IRewardReceiver.
      * Emits a Release event.
-     * @param subjectType agents, scanner or future types of stake subject. See SubjectTypes.sol
+     * @param subjectType type id of Stake Subject. See SubjectTypeValidator.sol
      * @param subject id identifying subject (external to FortaStaking).
      * @param account that staked on the subject.
      * @return available reward transferred.
      */
+    /*
```

Comment:
Let's delete all the code we're removing, instead of commenting it out.

Reply:
Done